getting-started/FAQ.md

```diff
@@ -2,42 +2,43 @@
+* The Pod Security Context expect the security profile to be specified in its native language, for instance, AppArmor profile for AppArmor. SELinux profile if SELinux is to be used. The profile language is extremely complex and this complexity could backfire i.e, it could lead to security holes.
+* Security Profile updates are manual and difficult: When an app is updated, the security posture might change and it becomes difficult to manually update the native rules.
+* No alerting of LSM violation on managed cloud platforms: By default LSMs send logs to kernel auditd, which is not available on most managed cloud platforms.
 
 KubeArmor solves all the above mentioned problems.
-* It maps YAML rules to LSMs (apparmor, bpf-lsm) rules so prior knowledge of different security context (native AppArmor, SELinux) is not required.  
-* It's easy to deploy: KubeArmor is deployed as a daemonset. Even when the application is updated, the enforcement rules are automatically applied.  
-* Consistent Alerting: KubeArmor handles kernel events and maps k8s metadata using ebpf.  
-* KubeArmor also runs in systemd mode so can directly run and protect Virtual Machines or Bare-metal machines too.  
-* Pod Security Context cannot leverage BPF-LSM at all today. BPF-LSM provides more programmatic control over the policy rules.  
+* It maps YAML rules to LSMs (apparmor, bpf-lsm) rules so prior knowledge of different security context (native AppArmor, SELinux) is not required.
+* It's easy to deploy: KubeArmor is deployed as a daemonset. Even when the application is updated, the enforcement rules are automatically applied.
+* Consistent Alerting: KubeArmor handles kernel events and maps k8s metadata using ebpf.
+* KubeArmor also runs in systemd mode so can directly run and protect Virtual Machines or Bare-metal machines too.
+* Pod Security Context cannot leverage BPF-LSM at all today. BPF-LSM provides more programmatic control over the policy rules.
 * Pod Security Context do not manage abstractions. As an example, you might have two nodes with Ubuntu, two nodes with Bottlerocket. Ubuntu, by default has AppArmor and Bottlerocket has BPF-LSM and SELinux. KubeArmor internally picks the right primitives to use for enforcement and the user do not have to bother explicitly stating what to use.
 </details>
 
-<details><summary><h4>What is visibility that I hear of in KubeArmor and how to get visibility information?</h4></summary>  
+<details><summary><h4>What is visibility that I hear of in KubeArmor and how to get visibility information?</h4></summary>
 
 KubeArmor, apart from been a policy enforcement engine also emits pod/container visibility data. It uses an eBPF-based system monitor which keeps track of process life cycles in containers and even nodes, and converts system metadata to container/node identities. This information can then be used for observability use-cases.
 
```

Comment:
Can you please change the `karmor log --json` to `karmor logs --json` on line 45?

Reply:
Great catch. Fixed